### PR TITLE
pkg/model pkg/diff pkg/client/:simplify code and remove unused code

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:#7800

### CODE REVIEW

File: pkg/model/network.go [hunk 1]
```diff
@@ -120,9 +120,8 @@ func (b *NetworkModelBuilder) Build(c *fi.ModelBuilderContext) error {
 			VPC:         b.LinkToVPC(),
 			DHCPOptions: dhcp,
 		})
-	} else {
-		// TODO: would be good to create these as shared, to verify them
 	}
+	// TODO: would be good to create these as shared, to verify them
```
Comment (rifelpet) [inline pkg/model/network.go]: I think putting this TODO comment above the `if !sharedVPC {` conditional that it was originally in might be more appropriate.

Comment (tanjunchen) [reply pkg/model/network.go]: Sure.this seems good to me.
I will update it.

Comment (rifelpet) [reply pkg/model/network.go]: ah, I meant above line 101. sorry. given theres no `else` case anymore, a comment related to an `else` case would probably be best above the `if` statement itself.

**Comment by tanjunchen** [comment]
done

**Comment by rifelpet** [comment]
thanks!

/lgtm
/approve

**Pull request merged**